### DISCUSSION TITLE
如何将musicxml文件转化为json格式？

### DISCUSSION THREAD

**Discussion by phpsxg** [CoderLine/alphaTab#661] [Q&A] [answered]
请问 如何将musicxml文件转化为json格式，谢谢

**Comment by Danielku15** [comment, marked as answer]
Most people will not be able to help you unless you try to write in English.

**Comment by lannka** [comment]
The question is: how to convert musicxml to json format.
It seems to be out of scope here. Might be a question relevant in a musicxml forum. 

同学，你的问题超纲了，更适合在musicxml的社区里提出

**Reply by hhly** [reply]
请问下 alphatab在safari和手机上没声音是啥问题，没有报错信息

**Reply by lannka** [reply]
happy to chat offline. VX: lankakaa

**Comment by hhly** [comment]
请问下 alphatab在safari和手机上没声音是啥问题，没有报错信息

**Reply by Danielku15** [reply]
Most people do not speak Chinese. You might get better help discussing in English.

**Reply by hhly** [reply]
What is the problem that alphatab has no voice on safari and mobile phones? There is no error message

**Reply by hhly** [reply]
<img width="439" alt="image" src="https://user-images.githubusercontent.com/29226405/199247333-98c536dd-18ec-4150-af1e-696d565f5e30.png">
I use this method to load the sound source. There is sound in other browsers, but no sound on safari and ios phones

**Reply by Danielku15** [reply]
Must be a bug in your software. Generally alphaTab is working fine on Safari and Mobile phones, I just lately [tested it through](https://github.com/CoderLine/alphaTab/pull/995#issuecomment-1272565179) in the 1.2.3 release .
 You could increase the [`logLevel`](https://alphatab.net/docs/reference/settings/core/loglevel) to find out if something is happening inside alphaTab. 

You might want to double check if the URL you are loading there is really available or there is an error. Additional events which give you the deeper insights: 
* https://alphatab.net/docs/reference/api/soundfontload/ 
* https://alphatab.net/docs/reference/api/soundfontloaded/
* https://alphatab.net/docs/reference/api/error/

Maybe you want to double check if your application is working on Desktop and debug some things there if data is loaded and working.

**Reply by hhly** [reply]
<img width="1415" alt="image" src="https://user-images.githubusercontent.com/29226405/199635545-4b25b7a6-f2d3-4e56-a96a-7ce12c384828.png">
I compared the logs of Chrome and Safari, and found that the audio source was not loaded successfully and there were no errors in Safari

**Reply by Danielku15** [reply]
Have you checked the "Network" tab to see whether the download is actually done by the browser? It looks to me that the browser on the right cannot load the file from the server, there is no download progress reported. 

Maybe you can share a running version of your app and the exact browser/device you are using in your tests? Just from screenshots finding the root cause will be time consuming.
